applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -87,15 +91,40 @@ static struct bt_conn_cb dfu_conn_callbacks = {
 	.disconnected = dfu_disconnected_cb,
 };
 
-static void on_ble_core_ready_dfu_entry(void)
+static void dfu_set_bt_name(void)
 {
-	char name[CONFIG_BT_DEVICE_NAME_MAX];
+	char name[CONFIG_BT_DEVICE_NAME_MAX] = {0};
+
+	strlcpy(name, CONFIG_BT_DEVICE_NAME, CONFIG_BT_DEVICE_NAME_MAX);
+	strlcat(name, "_", CONFIG_BT_DEVICE_NAME_MAX);
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+	strlcat(name, GW_TAG, CONFIG_BT_DEVICE_NAME_MAX);
+#else
+	int ret;
```

Comment:
Nitpick: These should be declared on the top of the function

Reply:
If CONFIG_AUDIO_DEV is GATEWAY, ret and channel will become unused variables.